tests/test_jmcomic/test_jm_feature.py

```
@@ -0,0 +1,200 @@
+
+        # 直接使用测试环境配置的下载目录
+        export_dir = self.option.dir_rule.base_dir
+
+        # 定义导出路径指向测试目录
+        f_pdf = Feature.export_pdf(pdf_dir=export_dir)
+        f_zip = Feature.export_zip(zip_dir=export_dir)
+        f_long_img = Feature.export_long_img(img_dir=export_dir)
+
+        # 组合下载并导出
+        combo = f_pdf + f_zip + f_long_img
+        album, dler = jmcomic.download_album(album_id, self.option, extra=combo)
```

Ruff (0.15.12)

[warning] Unpacked variable `dler` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)